package.json

```diff
@@ -6,19 +6,50 @@
   "dependencies": {
     "a-sync-waterfall": "^1.0.0",
     "asap": "^2.0.3",
+    "postinstall-build": "^5.0.1",
     "yargs": "^3.32.0"
   },
   "browser": "./browser/nunjucks.js",
   "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-plugin-istanbul": "^4.1.5",
+    "babel-plugin-module-resolver": "3.0.0-beta.5",
```

Comment:
Using custom resolver is close to cardinal sin. It will make code unusable for others, if they, for instance, will decide to require `src` directly. Unless it's used for something else.

Reply:
Definitely, the aliasing functionality of this plugin is a terrible idea, but in my case I'm using it with webpack to remap certain packages for web (specifically: `src/loaders.js` becomes `src/web-loaders.js` and `src/precompiled-loaders.js` for full and slim builds, respectively), and also to map modules unneeded by the slim build to an empty module (e.g. `src/precompile.js`)

This config is specified in [scripts/bundle.js](https://github.com/mozilla/nunjucks/blob/d94db12c3222c4cbe37411f160cad6541bf2712a/scripts/bundle.js#L46-L62) and only applies to the browser bundles.

Currently on master this happens in bin/bundle.js, except it uses `webpack.NormalModuleReplacementPlugin` instead. I found that using that with babel caused weirdness in the code coverage results, and anyway I like the flexibility of being able to do browser / slim mappings in a callable.

Reply:
That's a great idea. For whatever reason using separate entrypoints had not occurred to me, probably because I was thinking about how it used to be done, but it's a much more elegant solution (and would make jsdom-style testing easier too)